src/pyfmi/fmi.pyx

```diff
@@ -7002,7 +7002,7 @@ cdef class FMUModelCS2(FMUModelBase2):
                 raise InvalidVersionException("The FMU could not be loaded. This class only supports FMI 2.0 for Co-simulation.")
 
         if self.get_capability_flags()['needsExecutionTool']:
-            raise FMUException('Models that need an execution tool are not supported')
+            raise FMUException('The FMU specifies that it requires an external execution tool to simulate, this is not supported.')
```

Comment:
```suggestion
            raise FMUException('The FMU specifies that it requires an external execution tool ("needsExecutionTool=True") to simulate, this is not supported.')
```
I think a concrete pointer to this flag could be useful as well here, then one has a starting point for googling.

Reply:
I was considering adding that as well, I just wonder if people are aware of the different flags within the FMU.
How about the following?
"The FMU specifies 'needsExecutionTool=true' which implies that it requires an external execution tool to simulate, this is not supported."

Reply:
Updated